rs/rosetta-api/tests/system_tests/common/system_test_environment.rs

```diff
@@ -0,0 +1,126 @@
+use crate::common::{
+    constants::{DEFAULT_INITIAL_BALANCE, STARTING_CYCLES_PER_CANISTER},
+    utils::test_identity,
+};
+use candid::{Encode, Principal};
+use ic_agent::Identity;
+use ic_icp_rosetta_client::RosettaClient;
+use ic_icp_rosetta_runner::{start_rosetta, RosettaContext, RosettaOptionsBuilder};
+use ic_icrc1_test_utils::minter_identity;
+use ic_ledger_test_utils::build_ledger_wasm;
+use ic_ledger_test_utils::pocket_ic_helpers::ledger::LEDGER_CANISTER_ID;
+use ic_rosetta_test_utils::path_from_env;
+use ic_types::PrincipalId;
+use icp_ledger::{AccountIdentifier, LedgerCanisterInitPayload};
+use pocket_ic::{nonblocking::PocketIc, PocketIcBuilder};
+use std::collections::HashMap;
+use tempfile::TempDir;
+
+pub struct RosettaTestingEnvironment {
+    pub _pocket_ic: PocketIc,
+    pub _rosetta_context: RosettaContext,
+    pub rosetta_client: RosettaClient,
+}
+
+impl RosettaTestingEnvironment {
+    pub fn builder() -> RosettaTestingEnviornmentBuilder {
+        RosettaTestingEnviornmentBuilder::new()
+    }
+}
+
+pub struct RosettaTestingEnviornmentBuilder {
+    pub persistent_storage: bool,
```

Comment:
Is it used anywhere?

Reply:
Oh, this is a draft, nevermind :)
Still haven't found a good way to search for review PRs in github :)

Reply:
it is used, because when we create the rosetta instance the default option is to create it in memory. It leaves the option to also create it using persistent storage.

Reply:
Do you mean you plan to use it in the future? I synced your branch, removed the variable and the setter and everything builds...

Reply:
I mean it is not read anywhere. Please add tests that use all the setter methods or remove the setters for now.

Reply:
I removed them